frontend/src/metabase/common/components/FilterPicker/FilterOperatorPicker.tsx

```diff
@@ -0,0 +1,59 @@
+import type { ChangeEvent } from "react";
+import { useMemo } from "react";
+import Select from "metabase/core/components/Select";
+import * as Lib from "metabase-lib";
+
+type Option = {
+  name: string;
+  value: Lib.FilterOperatorName;
+};
+
+type FilterOperatorPickerProps = {
+  query: Lib.Query;
+  stageIndex: number;
+  column: Lib.ColumnMetadata;
+  value: Lib.FilterOperatorName | null;
```

Comment:
It would be better if we could accept `Lib.FilterOperator` here

Reply:
even after https://github.com/metabase/metabase/pull/34338 ?

We always get the available operators from `filterableColumnOperators`, but we can more easily (and performantly) pass around and compare short names after that.